app/api/player/$id.mjs

```diff
@@ -5,9 +5,12 @@ export const get = [async function get (req) {
   const { session: prevSession } = req
   const { id } = req.pathParameters
 
-  // `id` formatted as 'N1-N2' where N1 is the single digit album ID,
-  // and N2 is the single digit track number
-  const [albumId, , trackId] = id
+  // `id` formatted as 'N1-N2' where N1 is the numeric album ID,
+  // and N2 is the numeric track number
+  const [, albumId, trackId] = /([0-9]+)-([0-9]+)/.exec(id)
```

Comment:
🔥 👍🏻

Reply:
Apparently we need to make albums with more tracks @colepeters 😂